src/apps/loyalty/server/middlewares/session.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from "express"
```

Comment:
Yeah a rename is probably a good call. Maybe something like `TrackSessionIDMiddleware`?